### DISCUSSION TITLE
NEW - WORKING - WEBVIEW (WORKS ON ALL VERSIONS!!)

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1123] [Webview JOIN TITANIUM NETWORK TO SHARE PLEASE]
a WebView exploit that uses the "add person" button on signin

click on "add person" on signin (if u dont have this, it wont work for u)
click on "signin options" then click "signin withh GitHub"
scroll down and click "terms" (Hyperlink)
then scroll down again and press "pricing" (Hyperlink)
when on the pricing page click the github logo
you will be on the github main page
click on "open-source" then "topics"
scroll down and click "load more" then do it agin
once you have loaded more twice - you should see "Google" appear
click on google then click the link to google.com
Voila!

I call it :Rizz-View
View the full writeup here: https://github.com/Egduffuu/RizzView

**Comment by tazz-devv** [comment]
this has been patched lmao

**Reply by legendaryfishwastaken** [reply]
no it hasn't

**Comment by akabutnicer** [comment]
who's going to tell bro
